src/oncall/user_sync/slack.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import time
+from gevent import sleep
+import logging
+
+from phonenumbers import format_number, parse, PhoneNumberFormat
+from slackclient import SlackClient
+from oncall import db
+
+logger = logging.getLogger(__name__)
+
+
+def normalize_phone_number(num):
+    return format_number(parse(num.decode('utf-8'), 'US'), PhoneNumberFormat.INTERNATIONAL)
+
+
+def fetch_oncall_usernames(connection):
+    cursor = connection.cursor()
+    cursor.execute('SELECT `name` FROM `user`')
+    users = [row[0] for row in cursor]
```

Comment:
Maybe do a set comprehension here to avoid cast later ? Not sure if that'd be better or worse